bot.js

```diff
@@ -100,3 +33,32 @@ bot.on('message', (msg) => {
 });
 
 bot.login(token);
+
+function debugRequire(npmPackage, errorMessage) {
+	try {
+		require(npmPackage);
```

Comment:
There are weird tab/space issues in here that I'll fix in the morning. Also I think this should be "return require(npmPackage)". Normally the return is unnecessary (since it's the last executed statement in the function), but apparently require is a special case.